sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/MethodOverloadsShouldBeGrouped.cs

```diff
@@ -113,5 +63,10 @@ protected override void Initialize(SonarAnalysisContext context)
             SyntaxKind.InterfaceBlock,
             SyntaxKind.StructureBlock);
         }
+
+        private bool IsStaticStatement(MethodBaseSyntax statement) => statement.DescendantTokens().Any(x => x.Kind() == SyntaxKind.SharedKeyword);
+
+        private bool IsMustInheritStatement(MethodBaseSyntax statement) => statement.DescendantTokens().Any(x => x.Kind() == SyntaxKind.MustOverrideKeyword);
```

Comment:
_nitpick_ - would it be clearer to name this `IsAbstract` ? afterall, semantically it would be the same, right? it would make the code more uniform and readable